README.md

```diff
@@ -36,6 +36,11 @@ This example requires access to the docker images of observatory control system
 `docker-compose up -d` from within this directory. Once running, you should be able to use the credentials given above to access the admin interface of any of the applications, or to login and view/submit requests on the Observation Portal's web interface.
 
 By default, the docker-compose file has ports set up such that:
-The Observation Portal should be accessible from <http://127.0.0.1:8000>
-The Configuration Database should be accessible from <http://127.0.0.1:7000>
-The Downtime Database should be accessible from <http://127.0.0.1:7500>
+
+* The **Web Interface** should be accessible from <http://127.0.0.1:8080>
+
+* The **Observation Portal** should be accessible from <http://127.0.0.1:8000>
```

Comment:
It might be worth adding in the /admin or /api on the end of the Observation Portal URL since just going to the base URL won't get you anywhere.